### DISCUSSION TITLE
Problema con App HomeAssistant

### DISCUSSION THREAD

**Discussion by bondeste** [indomus/forum#1568] [HUB: Home Assistant]
Ciao a tutti, ho dato da poco reinstallato HA su un pc win10 utilizzando Docker (so che non è la configurazione migliore, ma per il momento mi accontento).
La necessità di reinstallarlo è nata dal fatto che dopo un aggiornamento non mi andava più nulla.
Una volta installato il tutto e fatto la copia della precedente cartella di HA in quella nuova ho avuto qualche problema ma risolti quasi tutti.
Ora sto cercando di installare l'app di  HA sul mio smartphone Xiaomi ma nulla non riesco a connettermi. Una volta aperta l'app non mi riconosce in automatico l'istanza (sono collegato su wifi di casa) e anche provando ad inserire manualmente l'indirizzo che utilizzo per collegarmi da PC http://127.0.0.1:8123 mi dice che impossibile connettersi.
Ho controllato il file configuration.yaml e la riga `default_config`  è presente.
Ho provato a cercare informazioni in rete su problemi simile ma nulla, non riesco a  trovare nulla.
Voi avete qualche idea?
In passato ho avuto su un altro telefono l'app installata senza problemi.
Ho anche eliminato l'integrazione e entità del precedente dispositivo ma non è cambiato nulla. Ho anche pensato che magari nel copiare le cartelle della precedente versione in quella nuova mi sono portato dietro qualcosa che possa andare  in conflitto, ma sinceramente non saprei dove e cosa cercare.
grazie mille in anticipo dell'aiuto.

**Comment by gio-dot** [comment]
https://it.wikipedia.org/wiki/Localhost

**Comment by bondeste** [comment]
ho risolto. il tuo link mi ha fatto pensare che, visto che HA gira su Docker, l'indirizzo da inserire non fosse quello ma quello del Docker.
Infatti andando a consultare file HOST ho scoperto l'indirizzo del Docker e utilizzandolo nell'app ha funzionato.
Non so se fosse quello che intendevi ma cmq  grazie mille per la dritta.
A sto punto però non capisco come mai non devo comunque utilizzare l'indirizzo del docker anche da broswer???

**Reply by indomus** [reply]
127.0.0.1 è [l'indirizzo di loopback](https://it.wikipedia.org/wiki/Localhost), non di Docker. L'indirizzo locale della rete Docker, inoltre, non ti interessa.
Devi capire sulla rete LAN qual è l'indirizzo col quale HA è esposto (e ovviamente aver certezza che il firewall Windows non interferisca). Eg. 192.168.1.45
